independent-projects/resteasy-reactive/server/runtime/src/main/java/org/jboss/resteasy/reactive/server/injection/ContextProducers.java

```diff
@@ -133,6 +133,10 @@ SecurityContext securityContext() {
     }
 
     private ResteasyReactiveRequestContext getContext() {
-        return CurrentRequestManager.get();
+        ResteasyReactiveRequestContext context = CurrentRequestManager.get();
+        if (context == null) {
+            throw new IllegalStateException("No RESTEasy Reactive request in flight");
```

Comment:
```suggestion
            throw new IllegalStateException("No RESTEasy Reactive request in-flight");
```
See https://cayceberryman.com/hyphens-and-dashes-hyphens/

or maybe more directly: 

```suggestion
            throw new IllegalStateException("No RESTEasy Reactive request in progress");
```